src/collections/blog/2022/2022-12-06-k8s-1.26-release/index.mdx

```diff
@@ -128,8 +128,8 @@ Consistent to the Kubernetes API lifecycle is deprecations and removals of APIs
 
 Kubernetes is an ever-evolving platform. For those of you running workloads on Kubernetes taking detailed note of API changes and enhancements is an important activity as you endevour to keep your clusters upgraded with release releases. A more secure, scalable, and flexible Kubernetes is our collective goal. Dign into more details about deprecation, removals, and the latest changes in the 1.26 [release notes](https://relnotes.k8s.io/).
 
-On behalf of the Layer5 community and all of the CNCF projects that its contributors steward, thank you to everyone who participated in this Kubernetes release, and congratulations! 
+On behalf of the Layer5 community and all of the CNCF projects that its contributors steward, thank you to everyone who participated in this Kubernetes release, and congratulations!
 
-As an end-to-end, open-source, multi-cluster Kuberentes management platform, Meshery makes Day 2 Kubernetes cluster management a breeze. Run Meshery to explore the behavorial changes of this Kubernetes release and what they really mean to you. 
+As an end-to-end, open-source, multi-cluster Kuberentes management platform, Meshery makes Day 2 Kubernetes cluster management a breeze. Run Meshery to explore the behavorial changes of this Kubernetes release and what they really mean to you.
```

Comment:
@Shivam-AfA will you remove the changes made to this file? They are formatting changes-only, correct? If so, let's not include them. It only makes reviewing take longer and adds burden to the history of tracking changes to the file over time.